Content.Server/Botany/Systems/MutationSystem.cs

```diff
@@ -39,40 +39,41 @@ public void MutateSeed(SeedData seed, float severity)
         const int totalbits = 270;
```

Comment:
fun fact - it should be 260 actually, there are 80 bits in "Fun".

Reply:
When I was looking at this, it in fact seemed like something was wrong with calculations. But I didn't change them.